kani-driver/src/call_goto_instrument.rs

```diff
@@ -130,6 +134,16 @@ impl KaniSession {
         self.call_goto_instrument(args)
     }
 
+    fn goto_sanity_check(&self, file: &Path) -> Result<()> {
```

Comment:
This seems to be running it twice, when we're already doing it with the cbmc args?

Reply:
Yeah. I figured it's a cheap analysis, so we might as well run before any `goto-instrument` optimization and right before verification.

Reply:
that said, I totally misplaced it. Let me move it up.